certs/certs.c

```diff
@@ -51,7 +54,7 @@ const char certificates[] =
 "YSEY1QSteDwsOoBrp+uvFRTp2InBuThs4pFsiv9kuXclVzDAGySj4dzp30d8tbQk\r\n"
 "CAUw7C29C79Fv1C5qfPrmAESrciIxpg0X40KPMbp1ZWVbd4=\r\n"
 "-----END CERTIFICATE-----\r\n"
-/*D-TRUST Root Class 3 CA 2 2009*/
+/*D-TRUST Root Class 3 CA 2 2009 --Europe--*/
```

Comment:
>Europe [](start = 35, length = 6)

i don't think this is exactly true. i think this is particular to Germany and not necessarily all of Europe. we might need to double check though. [Link](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud)

Reply:
Fixed. And yeah, it does appear to be Germany specific.

Reply:
I'd suggest being even more specific: something like Azure Germany Cloud (codiname Black Forest).

The same for Azure China Cloud (codiname Mooncake).

---
In reply to: [336702649](https://github.com/Azure/azure-iot-sdk-c/pull/1274#discussion_r336702649) [](ancestors = 336702649)